specification/app/resource-manager/Microsoft.App/preview/2023-05-02-preview/examples/Patches_Apply.json

```diff
@@ -0,0 +1,69 @@
+            "name": "testPatch-25fe4b",
+            "type": "Microsoft.App/builders/patches",
+            "properties": {
+              "provisioningState": "Succeeded",
+              "detectionStatus": "Succeeded",
+              "patchStatus": "Succeeded",
+              "patchDetails": {
+                "targetImageName": "testregistry.azurecr.io/test-image:latest",
+                "newRunImage": "mcr.microsoft.com/oryx/builder:run-dotnet-aspnet-7.0.7-cbl-mariner2.0",
+                "oldRunImage": "mcr.microsoft.com/oryx/builder:run-dotnet-aspnet-7.0.5-cbl-mariner2.0",
+                "patchType": "RunImageSecurity",
+                "targetEnvironmentId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg/providers/Microsoft.App/managedEnvironments/test-env",
+                "targetContainerAppId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg/providers/Microsoft.App/managedEnvironments/test-env/apps/test-app",
+                "targetContainerName": "test-container"
+              }
+            },
+            "systemData": {
+              "createdBy": "sample@microsoft.com",
+              "createdByType": "User",
+              "createdAt": "2022-10-11T11:05:51.4940669Z",
+              "lastModifiedBy": "sample@microsoft.com",
+              "lastModifiedByType": "User",
+              "lastModifiedAt": "2022-10-11T11:05:51.4940669Z"
+            }
+          }
+        }
+      },
+      "201": {
+        "headers": {
+          "Azure-AsyncOperation": "https://management.azure.com/subscriptions/{subscription-id}/providers/Microsoft.App/locations/{location}/operationStatuses/{operationId}"
+        },
+        "body": {
+          "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg/providers/Microsoft.App/builders/testBuilder/patches/testPatch/testPatch-25fe4b",
+          "name": "testPatch-25fe4b",
+          "type": "Microsoft.App/builders/patches",
+          "properties": {
+            "provisioningState": "Succeeded",
+            "detectionStatus": "Succeeded",
+            "patchStatus": "InProgress",
+            "patchDetails": {
+              "targetImageName": "testregistry.azurecr.io/test-image:latest",
+              "newRunImage": "mcr.microsoft.com/oryx/builder:run-dotnet-aspnet-7.0.7-cbl-mariner2.0",
+              "oldRunImage": "mcr.microsoft.com/oryx/builder:run-dotnet-aspnet-7.0.5-cbl-mariner2.0",
+              "patchType": "RunImageSecurity",
+              "targetEnvironmentId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg/providers/Microsoft.App/managedEnvironments/test-env",
+              "targetContainerAppId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg/providers/Microsoft.App/containerApps/test-app",
+              "targetContainerName": "test-container"
+            }
+          }
```

Comment:
I think cormac mentioned this previously, but lets add the system data in the async operation as well